test/html.test.js

```diff
@@ -665,3 +672,88 @@ QUnit.test('does not wrap mp4 source buffers', function() {
   );
   QUnit.equal(mediaSource.sourceBuffers.length, 2, 'created native buffers');
 });
+
+QUnit.test('can get activeSourceBuffers', () => {
+  let mediaSource = new videojs.MediaSource();
+
+  QUnit.ok(Array.isArray(mediaSource.activeSourceBuffers));
```

Comment:
It's kinda funny, but that was exactly why I used Array.isArray in this test. I probably should've made a comment or added it to the test title, but since we are going with the simplified route of using an array instead of a SourceBufferList, I wanted to put it in the tests as a kind of "documentation" to show that we meant to do that and have assumptions based on it.